ParticleSpeciesNames.md

```diff
@@ -0,0 +1,47 @@
+Convention for Specifying Particle Species 
+------------------------------------------
+
+This convention is for standardizing the names of particle species.
+
+SubAtomic Particle Names
+----------------------------------
+
+ - Subatomic particle names:
+  - antimuon
+  - antiproton
+  - bottom
+  - charm-antiquark
+  - charm-quark
+  - deuteron
+  - down-antiquark
+  - down-quark
+  - electron
+  - electron-neutrino
+  - gluon
+  - graviton
+  - higgs-boson
+  - muon
+  - muon-neutrino
+  - neutron
+  - photon
+  - pion
+  - positron
+  - proton
+  - strange-antiquark
+  - strange-quark
+  - tao-antiquark
+  - tao-quark
+  - tao-neutrino
+  - top-antiquark
+  - top-quark
+  - up-antiquark
+  - up-quark
+  - w-boson
+  - z-boson
+
+
+Names for Atoms and Molecules
+-------------------
+
+ - Atoms and molecules: Use standard chemical notation. Eg: "H20". Isotopes are denoted by a pound symbol "#" followed by the isotopic number followed by the chemical symbol. Eg: "#3He" for Helium-3.
```

Comment:
maybe that's a stupid question, but is there some kind of ISO naming already for this on which we can rely on?

Reply:
I looked but I could not find anything unfortunately.

Reply:
I see, thanks for checking.
And something from PDG (the particle data group), maybe? Just that we don't miss something that already defines spellings.

Reply:
(sides of that insecurity of mine I like the proposed spellings :-) )

Reply:
I had looked at the PDG as well without luck.